plugin/loadbalance/loadbalance_test.go

```diff
@@ -24,6 +24,8 @@ func TestLoadBalance(t *testing.T) {
 		addressExtra  int
 		mxAnswer      int
 		mxExtra       int
+		nsAnswer      int
```

Comment:
please write a new test function that just tests this shuffle

Reply:
What is the new test function going to test that is not covered in the table test ?